### ISSUE TITLE
NeRF

### ISSUE THREAD

**Issue by StrongLachlanM** [StrongResearch/isc-demos#37]
### Source / repo
https://github.com/yenchenlin/nerf-pytorch
https://www.matthewtancik.com/nerf

### Model description
NeRF

### Dataset
[]

### Literature benchmark source
https://arxiv.org/abs/2003.08934

### Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

### Strong Compute result achieved
[VALUE/S]

### Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: Adam

### Logs gist
[URL]
